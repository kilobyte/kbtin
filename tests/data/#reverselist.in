#reversel r a b c
#showme $r
#reversel r {{a b} {c d} e}
#showme $r
