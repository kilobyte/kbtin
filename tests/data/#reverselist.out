c b a
e {c d} {a b}
